reflect_test.go

```diff
@@ -68,6 +80,11 @@ type TestUser struct {
 	Feeling ProtoEnum `json:"feeling,omitempty"`
 	Age     int       `json:"age" jsonschema:"minimum=18,maximum=120,exclusiveMaximum=true,exclusiveMinimum=true"`
 	Email   string    `json:"email" jsonschema:"format=email"`
+
+	TestEnumVal    ABCEnumVal  `json:"test_enum_val,omitempty"`
```

Comment:
I don't mind this approach, but I'm still not sure why not just support a `jsonschema:"enum=a,b,c"` tag parameter?

Reply:
In my case code duplication. I already have lists of enums to handle JSON marshall/unmarshalling for various types so a approach like the removes the need to keep the struct tags in sync.